frontend/src/scenes/session-recordings/player/list/PlayerList.tsx

```diff
@@ -74,7 +74,7 @@ export function PlayerList<T extends Record<string, any>>({
 
     return (
         <div className="PlayerList">
-            {sessionEventsDataLoading || sessionPlayerMetaDataLoading ? (
+            {!data.length && (sessionEventsDataLoading || sessionPlayerMetaDataLoading) ? (
```

Comment:
I think we can remove `sessionPlayerMetaDataLoading` here to show events data if that's fetched first?

Reply:
I don't think we can. The sessionEventsData load is triggered _after_ the metadata is loaded so we have to check for both otherwise you get an initial flash of the empty state before the metadata is loaded

Reply:
Ah you're right. I was thinking we could load events async but I see that the the events call depends on metadata we fetch first!